src/traces/scattergeo/attributes.js

```diff
@@ -64,6 +65,9 @@ module.exports = overrideAll({
             'these elements will be seen in the hover labels.'
         ].join(' ')
     }),
+    texttemplate: texttemplateAttrs({editType: 'plot'}, {
+        keys: ['latlon', 'text']
```

Comment:
Hmm. This should be 

```js
keys: ['lat', 'lon', 'location', 'text']
```

similar to the `hovertemplate` keys

Reply:
Fixed by 0b516f7086c2fff2e9e4009234cc804169eb2aa1 and tested in https://github.com/plotly/plotly.js/commit/7d216806f075a938bc0fbdbd9149ee09c4bac027